spec/features/admin/groups/create_spec.rb

```diff
@@ -0,0 +1,28 @@
+RSpec.describe 'Groups CREATE' do
+  before do
+    assume_admin_logged_in
+    visit '/admin/groups/new'
+  end
+
+  context 'invalid input' do
+    it 'validates presence' do
+      fill_in 'Name',     with: ''
+      click_button 'Create Group'
+
+      expect_an_error group_name:     :blank
+      expect_an_error group_resource: :blank
+    end
+  end
+
+  context 'valid input' do
+    it 'create new group' do
+      fill_in 'Name', with: 'SuperGroup'
```

Comment:
what about using variables instead of copy-pasting text? I mean we are filling in text field with some text and expecting that value on the page. why do you prefer this style ?